icu4j/main/classes/core/src/com/ibm/icu/message2/IdentityFormatterFactory.java

```diff
@@ -0,0 +1,51 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.message2;
+
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Creates a {@link Formatter} that simply returns the String non-i18n aware representation of an object.
+ */
+class IdentityFormatterFactory implements FormatterFactory {
+    /**
+     * {@inheritDoc}
+     *
+     * @internal ICU 72 technology preview
+     * @deprecated This API is for ICU internal use only.
+     */
+    @Deprecated
+    @Override
+    public Formatter createFormatter(Locale locale, Map<String, Object> fixedOptions) {
```

Comment:
Could you please remove the internal/deprecated annotations also from methods that are no longer public because their classes no longer are?

Reply:
Done.